### ISSUE TITLE
Understanding Status   Changes

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#359]
_From @awkawk on March 2, 2018 15:25_

<p>The <a   href="https://www.w3.org/TR/WCAG21/#status-changes"   target="_blank">3.2.6 Status Changes (AA) SC</a> has draft   Understanding content that can be viewed at <a   href="https://rawgit.com/w3c/wcag21/status-messages/understanding/21/status-messages.html"   target="_blank">Understanding Status   Changes</a>.</p><p>Please review the Understanding document   and submit a comment on this issue for any changes that are needed to the   content.</p>

_Copied from original issue: w3c/wcag21#795_

**Comment by michael-n-cooper** [comment]
_From @guyhickling on March 22, 2018 17:12_

Will changes made now to these Understanding documents show immediately in these URLs you have given, or will we have to wait until the next official release to see them?

**Comment by michael-n-cooper** [comment]
_From @alastc on April 17, 2018 9:1_

Hi @guyhickling, they will update whenever something is merged into the specific 'understanding' branch, which will generally be well before the next official release.

Most of the work Feb/March has been on testing implementations, the understanding docs & techniques will get the focus Apr - May.

HTH,

-Alastair

**Comment by michael-n-cooper** [comment]
_From @bruce-usab on April 17, 2018 18:31_

On the call today I volunteered to provide and editorial review.  I have attached an MS Word version showing track changes with my suggested edits.  That is convenient way to work for me, but does it help whoever has lead on this?  I will need some hand-holding to do a pull request instead.  Also I noted some questions.

- The current version has the @@@ which will need a good bit of work.
- The current version kind of intermixes use of page, screen, and viewport.  I think the choices need a little bit more review.

[understanding-sc-326-status-changes.docx](https://github.com/w3c/wcag21/files/1921101/understanding-sc-326-status-changes.docx)

**Comment by michael-n-cooper** [comment]
_From @alastc on April 18, 2018 16:19_

Hi @bruce-usab, 

Yes, I think that helps, thanks for the review. @DavidMacDonald and @mbgower were editing this previously, but anyone can jump onto edits.

If you do want to try, I think the easiest way is to open the code view of [that page in the status-changes branch](https://github.com/w3c/wcag21/blob/status-changes/understanding/21/status-changes.html), and either edit there, or copy into a text editor and copy back.

However, there are also some comments that would benefit from answsers by Michael/David.

Separately, we should also include the action from issue #843

> Any techniques that are created for status message to allow a means of suppressing status messages will need to do so in a very careful fashion, based on clear user preferences. As you note, it will likely require design attention to decide whether certain status messages should not be elevated to 'disruptive' messages (i.e., modal dialogs) in such a scenario.

**Comment by michael-n-cooper** [comment]
_From @mbgower on April 19, 2018 22:29_

I will incorporate the changes from @bruce-usab . I also want to do a quick pass through of the Understanding document, because the rewording to Status Messages (from Status Changes) slightly reduced the scope of the SC, and some of the more 'best practice' material may now not be appropriate.

**Comment by michael-n-cooper** [comment]
_From @mbgower on April 20, 2018 4:50_

Okay, I've incorporated Bruce's changes. https://github.com/w3c/wcag21/commit/b132964a7c292a164f8710395319a9908508532a
@bruce-usab , I slightly modified the first sentence because I found your new one was a bit hard to parse (a lot of possible verbs acting as nouns, etc). I marked most of your stuff resovled. i responded to a few things we might want to think about a bit more.
[understanding-sc-326-status-changes-mg.docx](https://github.com/w3c/wcag21/files/1930663/understanding-sc-326-status-changes-mg.docx)

I have to say, not so humbly, that all the work I did on this seems to have been worth it. I'm pretty happy with how it is looking :) Now @DavidMacDonald and @steverep can rip it to shreds :)

**Comment by michael-n-cooper** [comment]
_From @bruce-usab on April 20, 2018 12:18_

Thanks Mike, looks great!  Yes, your work on this was well worth it!

**Comment by michael-n-cooper** [comment]
_From @DavidMacDonald on April 21, 2018 16:42_

I've reviewed the document, made a few editorial changes while not changing the substance. It was in good shape thanks to @mbgower ... The current Branch["Status-Changes"](https://github.com/w3c/wcag21/tree/status-changes) was in conflict with the Master, so after completing edits I created a new branch ["status-messages"](https://github.com/w3c/wcag21/tree/status-messages) and copied the status-message file over there. I've made a pull request #872

**Comment by michael-n-cooper** [comment]
_From @lseeman on April 22, 2018 16:25_

getting a 404

**Comment by michael-n-cooper** [comment]
_From @alastc on April 22, 2018 22:9_

Fixed, I updated to link to David's new branch: https://rawgit.com/w3c/wcag21/status-messages/understanding/21/status-messages.html

**Comment by mbgower** [comment]
I believe all topics have been addressed.
